src/scripts.js

```diff
@@ -207,28 +206,40 @@ function openRecipeInfo(event) {
 function generateIngredients(recipe) {
   determineIfEnoughIngredients(recipe)
   return recipe.ingredients.map(i => {
-    return `${i.quantity.amount} ${i.quantity.unit} ${i.name}`
-  }).join("\n");
+    let quantity = domUpdates.formatQuantity(i.quantity.amount)
```

Comment:
I think both of these variables can be const but we can just change later

Reply:
I started out with const for them but it threw an error!